ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot

```diff
@@ -227,22 +227,22 @@ msgstr ""
 
 #: ckanext/ontario_theme/templates/home/help.html:43
 msgid ""
-"not listed in the beta catalogue, but can be found in the <a "
+"not be listed in the beta catalogue, but can be found in the <a "
 "href=\"http://www.ontario.ca/data\" target=\"_blank\">current catalogue</a>. "
 "While the beta catalogue is in its testing phase, it will have some, but not "
 "all, of the datasets from the data catalogue."
 msgstr ""
 
 #: ckanext/ontario_theme/templates/home/help.html:46
-msgid "Search the current data catalogue"
+msgid "<a href=\"http://www.ontario.ca/data\">Search the current data catalogue</a>"
 msgstr ""
 
 #: ckanext/ontario_theme/templates/home/help.html:49
-msgid "Learn more about the beta stage"
+msgid "<a href="\/about#beta\">Learn more about the beta stage</a>"
```

Comment:
This is throwing an error `ckanext-ontario_theme.pot (line 241): unescaped double quote found`